src/org/salesforce/apexdoc/ApexDoc.java

```diff
@@ -208,7 +210,7 @@ public static ClassModel parseFileContents(String filePath) {
                 }
 
                 // gather up our comments
-                if (strLine.startsWith("/*")) {
+                if (strLine.startsWith("/**")) {
```

Comment:
Unfortunately, our NPSP code has copyright headers that start many files like this:

/*
    Written by Salesforce Foundation
    This program is released under the GNU Affero General Public License, Version 3. http://www.gnu.org/licenses/
*/

the parser gets tripped up by seeing the word Public on line 3.  It needs to be smart enough to know it was in a comment block to avoid this.  Since the parser knew how to deal with blocks beginning with /** and avoided this issue, the simplest fix was to relax the constraint that the apexdoc block had to start with /** rather than just /*.

while I agree that /** is the correct syntax, changing it to enforce this also requires handling and ignoring normal comment blocks.  So I can't take this pull request or it will break parsing of NPSP (which is why I undertook updating this tool in the first place).